libvast/include/vast/msgpack_table_slice.hpp

```diff
@@ -46,8 +46,10 @@ class msgpack_table_slice final {
   /// @param slice The encoding-specific FlatBuffers table.
   /// @param parent The surrounding chunk.
   /// @param batch A pre-existing record batch. Must always be nullptr.
+  /// @param schema A VAST schema matching the record batch. Must always be none.
```

Comment:
Same deal as above, explain that this is optional.

Reply:
According to the method signature, it's not optional. I added that it _must be none_, as for the batch.